bin/pakiti-client

```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
-
-    # Red Hat packages
+	
+	#We check if we got called with svmon parameter
```

Comment:
I believe you're using tab's to indent the code, please stick to using spaces (as in the rest of the code)